lib/pure/math.nim

```diff
@@ -56,7 +56,7 @@ type
     fcNegInf     ## value is negative infinity
 
 proc classify*(x: float): FloatClass =
-  ## classifies a floating point value. Returns `x`'s class as specified by
+  ## Classifies a floating point value. Returns `x`'s class as specified by
```

Comment:
That's simply not how I read. Many people don't see to read that way either. The documentation is on another line (and is indented) which implies the start of a new sentence if you are reading it that way...